app/views/dashboard/show.html.erb

```diff
@@ -2,8 +2,19 @@
   <div class="govuk-grid-column-full">
```

Comment:
At some point we need to decide on a unified approach to this. For the admin, we're sending them to a totally different page. Maybe partials for each user type that drop in?